scripts/solve_network.py

```diff
@@ -1904,7 +1975,9 @@ def add_ci(n: pypsa.Network, year: str, config: dict, costs: pd.DataFrame) -> No
                 else 1,  # be conservative for nuclear (maintenance or unplanned shut downs)
                 carrier=generator,
                 efficiency=costs.at[generator, "efficiency"],
-                efficiency2=costs.at[carrier, "CO2 intensity"],
+                efficiency2= costs.at[carrier, "CO2 intensity"]
+                if carrier != "gas"
```

Comment:
```suggestion
                if generator != "allam"
```

not all "gas" generator have CCS characteristics, we can extend this once more technologies are included